src/Command/ConstantToValueCommand.php

```diff
@@ -0,0 +1,126 @@
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace PhpOptimizer\Command;
+
+use PhpOptimizer\Converter\ConstToValueConverter;
+use PhpOptimizer\PhpParser\NodeVisitor\ConstantIndexVisitor;
+use PhpOptimizer\PhpParser\NodeVisitor\ConverterVisitor;
+use PhpParser\Error;
+use PhpParser\NodeTraverser;
+use PhpParser\NodeVisitor\NameResolver;
+use PhpParser\ParserFactory;
+use PhpParser\PrettyPrinter\Standard as StandardPrettyPrinter;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class ConstantToValue.
+ *
+ * @author Loek van der Linde <lind0077@hz.nl>
+ */
+class ConstantToValueCommand extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setName('php-optimize:constant:tovalue')
+            ->setDescription('Converts all constants to their value')
+            ->addArgument(
+                'source_folder',
+                InputArgument::REQUIRED,
+                'Folder in which to run PHP-Optimize'
+            )
+            ->addArgument(
+                'build_folder',
+                InputArgument::REQUIRED,
+                'Folder in which to place optimized files'
+            )
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $directoryToScan = \getcwd() . '/' . $input->getArgument('source_folder');
+        $directoryToBuild = \getcwd() . '/' . $input->getArgument('build_folder');
+
+        $parser = (new ParserFactory())->create(ParserFactory::PREFER_PHP7);
+        $traverser = new NodeTraverser();
+
+        $constantsVisitor = new ConstantIndexVisitor();
+
+        $traverser->addVisitor(new NameResolver());
+        $traverser->addVisitor($constantsVisitor);
+
+        $files = new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($directoryToScan));
+        $files = new \RegexIterator($files, '/\.php$/');
+
+        foreach ($files as $file) {
+            try {
+                $code = \file_get_contents($file->getPathName());
+                $stmts = $parser->parse($code);
+
+                if (empty($stmts)) {
+                    continue;
+                }
+
+                $traverser->traverse($stmts);
+            } catch (Error $e) {
+                echo 'Parse Error: ', $e->getMessage();
+            }
+        }
+
+        $constants = $constantsVisitor->getConstants();
+
+        $convertor = new ConverterVisitor(new ConstToValueConverter($constants));
+        $traverser->addVisitor($convertor);
+
+        $prettyPrinter = new StandardPrettyPrinter();
+
+        // TODO Now we're looping twice, this should be easier to do
```

Comment:
For now looping twice is fine. Later we can index all the items we want to change and only loop over those again.